image/spec/v1.1.md

```diff
@@ -419,7 +419,6 @@ Here is an example history section:
     "empty_layer": true
   }
 ]</pre>
-    </dd>
```

Comment:
This `</dd>` should not be removed; the actual fix is to remove the blank lines at line 408 and 410 instead. Similar to what was done in https://github.com/moby/moby/pull/37051

```patch
diff --git a/image/spec/v1.1.md b/image/spec/v1.1.md
index de74d91a19..8d0ec892da 100644
--- a/image/spec/v1.1.md
+++ b/image/spec/v1.1.md
@@ -405,9 +405,7 @@ whitespace. It has been added to this example for clarity.
             filesystem).
           </li>
         </ul>
-
 Here is an example history section:
-
 <pre>"history": [
   {
     "created": "2015-10-31T22:22:54.690851953Z",
```